roda-ui/roda-wui/src/main/java/org/roda/wui/api/controllers/UserManagement.java

```diff
@@ -353,12 +353,14 @@ public static void requestPasswordReset(String servletPath, String usernameOrEma
     }
 
     User user = UserManagementHelper.requestPasswordReset(username, email);
-    // 20180112 hsilva: need to set ip address for registering the action
-    user.setIpAddress(ipAddress);
-    sendRecoverLoginEmail(servletPath, user, localeString);
+    if (user != null) {
+      // 20180112 hsilva: need to set ip address for registering the action
+      user.setIpAddress(ipAddress);
+      sendRecoverLoginEmail(servletPath, user, localeString);
 
-    // register action
-    controllerAssistant.registerAction(user, LogEntryState.SUCCESS, RodaConstants.CONTROLLER_USER_PARAM, user);
+      // register action
+      controllerAssistant.registerAction(user, LogEntryState.SUCCESS, RodaConstants.CONTROLLER_USER_PARAM, user);
```

Comment:
check with the team if it is necessary to register an action even when the user does not exist